code/game/mecha/mecha.dm

```diff
@@ -6,6 +6,7 @@
 
 #define MELEE 1
 #define RANGED 2
+#define BOTH 4	//for mech equipment with effects depending on whether the target is adjacent
```

Comment:
You mean (MELEE|RANGED)?
This would also let you remove the BOTH check from the previous code.